storm.go

```diff
@@ -152,12 +153,26 @@ func toBytes(key interface{}, codec codec.MarshalUnmarshaler) ([]byte, error) {
 	if key == nil {
 		return nil, nil
 	}
-	if k, ok := key.([]byte); ok {
-		return k, nil
-	}
-	if k, ok := key.(string); ok {
-		return []byte(k), nil
+	switch t := key.(type) {
+	case []byte:
+		return t, nil
+	case string:
+		return []byte(t), nil
+	case int:
```

Comment:
I suspect you can collapse all the int cases into one line `case int, int64 ...:`. You should also consider adding the complete list.